components/tools/OmeroPy/src/omero/plugins/chown.py

```diff
@@ -65,10 +75,65 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Chown2
 
+    def _configure(self, parser):
+        parser.set_defaults(func=self.main_method)
+        self._add_wait(parser, default=-1)
+        parser.add_argument(
+            "--include",
+            help="Modifies the given option by including a list of objects")
+        parser.add_argument(
+            "--exclude",
+            help="Modifies the given option by excluding a list of objects")
+        parser.add_argument(
+            "--ordered", action="store_true",
+            help=("Pass multiple objects to commands strictly in the order "
+                  "given, otherwise group into as few commands as possible."))
+        parser.add_argument(
+            "--list", action="store_true",
+            help="Print a list of all available graph specs")
+        parser.add_argument(
+            "--list-details", action="store_true", help=SUPPRESS)
+        parser.add_argument(
+            "--report", action="store_true",
+            help="Print more detailed report of each action")
+        parser.add_argument(
+            "--dry-run", action="store_true",
+            help=("Do a dry run of the command, providing a "
+                  "report of what would have been done"))
+        parser.add_argument(
+            "--force", action="store_true",
+            help=("Force an action that otherwise defaults to a dry run"))
+        self._pre_objects(parser)
+        parser.add_argument(
+            "obj", nargs="*", type=GraphArg(self.cmd_type()),
+            help="objects to be processed in the form <Class>:<Id>"
```

Comment:
It's really unfortunate that you have to copy and paste all the code above just to change a little bit of the help text. Does argparse let you have the superclass one run but then *change* the help text for a given parser argument? Or (just like with `_pre_objects`) could you just override just a small helper method that `GraphControl._configure` calls?